### ISSUE TITLE
bot can't send message

### ISSUE THREAD

**Issue by adrianeranzi** [nickoala/telepot#425]
Whenever I use this "bot.sendMessage(chat_id,'Text Here'). I get an error saying bot can't send messages to bots.

**Comment by FireRasPi** [comment]
Thats right. A Bot is not able to send a message or something else to another bot. Use the Telegram Cli with a phone number to communicate between two bots.
